djangocms_alias/admin.py

```diff
@@ -171,12 +171,12 @@ def _get_alias_usage_link(self, obj, request):
             {"url": url, "disabled": False},
         )
 
-    def _get_alias_site_category_change_link(self, obj, request):
+    def _get_change_alias_settings_link(self, obj, request):
         url = admin_reverse('{}_{}_change'.format(
             obj._meta.app_label, obj.alias._meta.model_name), args=(obj.alias.pk,)
         )
         return render_to_string(
-            "admin/djangocms_alias/icons/edit_alias.html",
+            "admin/djangocms_alias/icons/change_alias_settings.html",
```

Comment:
Could we have disabled=False as a param coming into this function? That way if someone needs it to be disabled, they won't have to override the entire method.